### ISSUE TITLE
Asset reservation errors in test

### ISSUE THREAD

**Issue by lalgee** [LibraryOfCongress/concordia#1444]
**What behavior did you observe? Please describe the bug**
When using Chrome if I go to transcribe or review an asset in the test environment, I see the error "Unable to reserve this page: error". Because of this I am unable to edit the text or utilize the save or submit transcriptions. I am able to edit or accept an asset.

**How can we reproduce the bug?**

1. https://crowd-test.loc.gov/
2. Find an asset to transcribe or review in any campaign
3. See error

**Got screenshots? This helps us identify the issue**
![image](https://user-images.githubusercontent.com/20258744/118027858-69d06800-b330-11eb-892a-ec30cfbb84b5.png)

**Desktop (please complete the following information):**

-   OS: Microsoft
-   Browser: Chrome (not seeing the error in Firefox)

**Additional context**
https://crowd-test.loc.gov/next-transcribable-asset/ and https://crowd-test.loc.gov/next-reviewable-asset/ produce a Bad Request (400) error, which is likely related.

**Comment by jkueloc** [comment]
@lalg Good morning.   I was able to modify/transcribe 2 test assets/imgs in test successfully this morning without issue including the one mentioned in the issue above.  'Find a new page -> ' after 'Save' returned a fresh transcribable asset/img.  After Accepting a page for Review, clicking 'Review a new page' in the pop-up dialog also returned a fresh page in need of review.   Going directly to the links above also were successful. 

I used a Chrome New Incognito Window to do the testing. 

I honestly don't yet fully understand the way caching works both via the browsers (trying to be increasingly efficient), and the caching of Concordia.  But it's clear to me there can be delays and mixed results at times.

It's good that it didn't happen Firefox.

Try testing again using a Chrome New Incognito Window.
